src/utils/template.js

```diff
@@ -99,12 +99,14 @@ function showStatsSection() {
 
   document.querySelector('#moviesCount .ns-title').textContent = chrome.i18n.getMessage('moviesCount');
   document.querySelector('#moviesCount .ns-title').setAttribute('aria-label', chrome.i18n.getMessage('moviesCount'));
+  document.querySelector('#moviesCount .ns-image').setAttribute('src', chrome.runtime.getURL("../images/movie.svg"));
```

Comment:
As the [docs ](https://developer.chrome.com/extensions/runtime#method-getURL) say, the path inside _getURL_ function should be relative to the extension install directory, so I will change this path and the other four

Reply:
ok, no problem ;)